golem_sci/blockshelper.py

```diff
@@ -6,10 +6,11 @@ class BlocksHelper:
     def __init__(self, sci: SmartContractsInterface) -> None:
         self._sci = sci
 
-    def get_first_block_after(self, timestamp: int) -> Block:
+    def get_latest_existing_block_at(self, timestamp: int) -> Block:
         """
         Returns block with smallest number for which
-        `block.timestamp > timestamp` is satisfied.
+        `block.timestamp > timestamp` is satisfied or if
```

Comment:
quite frankly, I would be for preserving the old name, but to return `None` in case where there is no block after timestamp, instead of returning "latest available"

Reply:
ok, as agreed offline, we would like to have "latest available"